tools/output/blocks.js

```diff
@@ -1,1094 +1,1094 @@
-            astral: '\uD83B[\uDE00-\uDEFF]'
-        },
-        {
-            name: 'InArabic_Presentation_Forms_A',
-            bmp: '\uFB50-\uFDFF'
-        },
-        {
-            name: 'InArabic_Presentation_Forms_B',
-            bmp: '\uFE70-\uFEFF'
-        },
-        {
-            name: 'InArabic_Supplement',
-            bmp: '\u0750-\u077F'
-        },
-        {
-            name: 'InArmenian',
-            bmp: '\u0530-\u058F'
-        },
-        {
-            name: 'InArrows',
-            bmp: '\u2190-\u21FF'
-        },
-        {
-            name: 'InAvestan',
-            astral: '\uD802[\uDF00-\uDF3F]'
-        },
-        {
-            name: 'InBalinese',
-            bmp: '\u1B00-\u1B7F'
-        },
-        {
-            name: 'InBamum',
-            bmp: '\uA6A0-\uA6FF'
-        },
-        {
-            name: 'InBamum_Supplement',
-            astral: '\uD81A[\uDC00-\uDE3F]'
-        },
-        {
-            name: 'InBasic_Latin',
-            bmp: '\0-\x7F'
-        },
-        {
-            name: 'InBassa_Vah',
-            astral: '\uD81A[\uDED0-\uDEFF]'
-        },
-        {
-            name: 'InBatak',
-            bmp: '\u1BC0-\u1BFF'
-        },
-        {
-            name: 'InBengali',
-            bmp: '\u0980-\u09FF'
-        },
-        {
-            name: 'InBhaiksuki',
-            astral: '\uD807[\uDC00-\uDC6F]'
-        },
-        {
-            name: 'InBlock_Elements',
-            bmp: '\u2580-\u259F'
-        },
-        {
-            name: 'InBopomofo',
-            bmp: '\u3100-\u312F'
-        },
-        {
-            name: 'InBopomofo_Extended',
-            bmp: '\u31A0-\u31BF'
-        },
-        {
-            name: 'InBox_Drawing',
-            bmp: '\u2500-\u257F'
-        },
-        {
-            name: 'InBrahmi',
-            astral: '\uD804[\uDC00-\uDC7F]'
-        },
-        {
-            name: 'InBraille_Patterns',
-            bmp: '\u2800-\u28FF'
-        },
-        {
-            name: 'InBuginese',
-            bmp: '\u1A00-\u1A1F'
-        },
-        {
-            name: 'InBuhid',
-            bmp: '\u1740-\u175F'
-        },
-        {
-            name: 'InByzantine_Musical_Symbols',
-            astral: '\uD834[\uDC00-\uDCFF]'
-        },
-        {
-            name: 'InCJK_Compatibility',
-            bmp: '\u3300-\u33FF'
-        },
-        {
-            name: 'InCJK_Compatibility_Forms',
-            bmp: '\uFE30-\uFE4F'
-        },
-        {
-            name: 'InCJK_Compatibility_Ideographs',
-            bmp: '\uF900-\uFAFF'
-        },
-        {
-            name: 'InCJK_Compatibility_Ideographs_Supplement',
-            astral: '\uD87E[\uDC00-\uDE1F]'
-        },
-        {
-            name: 'InCJK_Radicals_Supplement',
-            bmp: '\u2E80-\u2EFF'
-        },
-        {
-            name: 'InCJK_Strokes',
-            bmp: '\u31C0-\u31EF'
-        },
-        {
-            name: 'InCJK_Symbols_and_Punctuation',
```

Comment:
Turns out the canonical name is `InCJK_Symbols_And_Punctuation`, with a capital letter `A`. This is technically a breaking change.

Reply:
Can we make it preserve the pre-existing names, so as to avoid the breaking change for now? Of course, we should probably open a separate issue about using the canonical names.

Reply:
It's not a breaking change to change casing. Feel free to update it to the canonical name! Per Unicode's recommendations, XRegExp already treats Unicode property names as case insensitive, with any spaces, hyphens, and underscores ignored. See https://github.com/slevithan/xregexp/blob/263ad8ca9963d4b81d10583f6e1edb446a88ad26/src/addons/unicode-base.js#L35-L38

Reply:
Oh, right! Woohoo :) (I was thinking about native Unicode property escapes, where I explicitly decided *not* to normalize.)

Patch updated.

Reply:
Ooh, I wasn't aware of that. Nice!